### DISCUSSION TITLE
Consolidating or Separating Collections for User Created Projects?

### DISCUSSION THREAD

**Discussion by StepanKukharskiy** [pocketbase/pocketbase#3641] [Q&A] [answered]
Hi there! I have multiple users who create projects, with each project containing text files and images. Would it be appropriate to store all the data for these projects in a single collection? What if there hundreds of thousands of projects in this collection? Or should I create a separate collection for projects created by each individual user? Thanks for your help!

**Comment by ganigeorgiev** [comment, marked as answer]
It is OK to have a single "projects" collection with a single/multiple `relation` field pointing back to the "users" collection (_you can also enable the cascade delete option for the field so that when a user is deleted to automatically delete its assigned projects_).

>  What if there hundreds of thousands of projects in this collection?

It should be fine as long as you have the disk space for your data (_I'm currently working on a project where one of the collections has 200k+ records and the performance is OK so far_). If you hit a bottleneck with a query performance you can try to create indexes for the frequently filter/sort fields (_they can be created from the collection panel in the Admin UI in the section under the schema fields_).

**Reply by StepanKukharskiy** [reply]
Awesome! Thanks!
